Python/bytecodes.c

```diff
@@ -5399,32 +5445,78 @@ dummy_func(
         tier2 op(_COLD_EXIT, ( -- )) {
```

Comment:
Could we keep this unchanged, and do any tweaks for better tracing in another PR?

Reply:
The old code no longer makes sense/is really cumbersome with the new tracing style (deopt to tier 1 then begin tracing). So sadly I'd rather not :(.